README.md

```diff
@@ -43,5 +55,6 @@ export NODE_DRIVER_REGISTRAR_IMAGE=quay.io/openshift/origin-csi-node-driver-regi
 export LIVENESS_PROBE_IMAGE=quay.io/openshift/origin-csi-livenessprobe:latest
 
 # Deploy the operator and everything it needs
-oc apply -f manifests/
+# oc apply -f manifests/
```

Comment:
You should remove the whole section "*To run the latest build of the operator:*" and add a new one that uses manifests from https://github.com/openshift/cluster-storage-operator/tree/master/assets/csidriveroperators/aws-ebs. Would `kustomize` help with replacing the image names in CSO assets?

I am not sure anyone actually runs custom built images in a cluster from scratch, I edit Deployment created by CSO and change the image name there.